packages/hive/src/Adaptor.js

```diff
@@ -0,0 +1,154 @@
+import {
+  execute as commonExecute,
+  composeNextState,
+} from '@openfn/language-common';
+import { expandReferences } from '@openfn/language-common/util';
+
+import hive from 'hive-driver';
+
+const { TCLIService, TCLIService_types } = hive.thrift;
+const utils = new hive.HiveUtils(TCLIService_types);
+
+const logger = message => console.log(message);
```

Comment:
Really don't think we should be aliasing `console.log` like this. It doesn't really save much time or typing, it looks weird when you come into the adaptor for the first time, it doesn't match the signature of `@openfn/logger`, and it doesn't behave the same as `console.log` :thinking: 

Sorry to be a pain but can we remove it?

Reply:
no that's a good catch. @mtuchi , what was the intention here?

Reply:
@josephjclark i was testing `@openfn/logger` then i ran into export errors. So my intention was to keep `logger` as a pattern that can be used to standardize how we log stuff in our adaptors. 

It's fine we can remove it